packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -226,3 +222,13 @@ module.exports = {
     tls: 'empty'
   }
 };
+
+// @remove-on-eject-begin
+// Override babel-loader options to not read .babelrc.
+var babelLoader = module.exports.module.loaders.find(l => l.loader === 'babel');
+babelLoader.query = babelLoader.query || {};
+babelLoader.query.babelrc = false;
+// We embed the preset instead of referring to it because otherwise
+// `env` options in the preset do not get taken into account.
```

Comment:
Is that so? I've seen some presets that use `env`, e.g. babel-preset-react-native [here](https://github.com/facebook/react-native/blob/17219720e40498c7f27314ab9126531b9cb07c37/babel-preset/configs/main.js#L43-L47). I wonder how that works?